app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,47 @@
+class PaymentsController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+  load_resource :conference, find_by: :short_title
+  authorize_resource :conference_registrations, class: Registration
+
+  def index
+    @payments = current_user.payments
+  end
+
+  def new
+    @total_amount_to_pay = Ticket.total_price(@conference, current_user, paid: false)
+  end
+
+  def create
+    @payment = Payment.new(payment_params)
+    @total_amount_to_pay = Ticket.total_price(@conference, current_user, paid: false)
+
+    if @payment.valid? && @payment.purchase(current_user, @conference, price_in_cents) && @payment.save
+      update_paid_ticket_purchases(@conference, current_user, @payment)
+      redirect_to conference_conference_registrations_path(@conference.short_title), flash: { success: 'Thanks! You have purchased your tickets successfully.' }
+    else
+      render 'new'
+    end
+  end
+
+  private
+
+  def price_in_cents
+    (@payment.amount * 100).round
+  end
+
+  def update_paid_ticket_purchases(conference, user, payment)
+    paid_ticket_purchases = TicketPurchase.where(conference_id: conference.id,
```

Comment:
why is the name of this variable **paid**_ticket_purchases if the where has `paid: false`?